### ISSUE TITLE
Enhance a11y for `Motion`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5394]
name: Motion

### Description
6 accessibility violations were found in the `Motion`.

#### [Components / Motion / Scroll / Viewport](https://yamada-ui.github.io/yamada-ui/?path=/story/components-motion-scroll--viewport)

Received:
Scrollable region must have keyboard access (scrollable-region-focusable)

Fix any of the following:
  Element should have focusable content
  Element should be focusable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=axeAPI

#### [Components / Motion / Animation / Layout](https://yamada-ui.github.io/yamada-ui/?path=/story/components-motion-animation--layout)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Motion / Animation / GroupLayout](https://yamada-ui.github.io/yamada-ui/?path=/story/components-motion-animation--group-layout)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Motion / Animation / GroupLayout](https://yamada-ui.github.io/yamada-ui/?path=/story/components-motion-animation--group-layout)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Motion / Animation / GroupLayout](https://yamada-ui.github.io/yamada-ui/?path=/story/components-motion-animation--group-layout)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Motion / Animation / GroupLayout](https://yamada-ui.github.io/yamada-ui/?path=/story/components-motion-animation--group-layout)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
